src/benchmarks/micro/corefx/System.Linq/Perf.Linq.cs

```diff
@@ -145,6 +145,31 @@ public void Skip(int size, int iteration, Perf_LinqTestBase.WrapperType wrapType
         public void Take(int size, int iteration, Perf_LinqTestBase.WrapperType wrapType) 
             => Perf_LinqTestBase.Measure(_sizeToPreallocatedArray[size], wrapType, col => col.Take(size - 1), _consumer);
 
+#if NETCORE
```

Comment:
Can you replace this with `#if !NETFRAMEWORK`?

Reply:
Done